client/a8c-for-agencies/components/task-steps/style.scss

```diff
@@ -0,0 +1,120 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.task-steps {
```

Comment:
Following the [Calypso CSS guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md#nesting), we should aim for a nesting level of two. It may not be necessary to nest everything under `.task-steps`, as all child components are already specific (`.task-steps__*`).

Reply:
Totally agree. Fixed it here: https://github.com/Automattic/wp-calypso/pull/96443/commits/c67444553b6836c9662c244c1f48c364bf621126